src/Tools/Microsoft.CodeAnalysis.Toolset.Open/Targets/VSL.Settings.targets

```diff
@@ -118,9 +118,9 @@
     </Reference>
   </ItemGroup>
 
-  <PropertyGroup Condition="'$(OutDir)' == ''">
-    <!-- If we don't already have an OutDir, set one-->
-    <OutDir>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..\..\..\..\Binaries\$(Configuration)'))\</OutDir>
+  <PropertyGroup>
+    <OutDir Condition="'$(OutDire)' == ''">$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..\..\..\..\Binaries\$(Configuration)'))\</OutDir>
```

Comment:
Very, because it means you probably went through the effort of typing it out again rather than just yank->paste ;)